go/Makefile

```diff
@@ -0,0 +1,21 @@
+NAME := checkplease
```

Comment:
I generally run:

```sh
make pi-run
```

This will end up rebuilding the binary, copying it over to the pi using `scp` and then running it in place.